examples/image_classification/README.md

```diff
@@ -0,0 +1,171 @@
+# Kaggle Benchmark by Autogluon 
+| Datset | GluonCV Baseline/Autogluon/1st/Rank | Search Space(net.choice/learning_rate/momentum/wd) | Configuration(epochs/trials/batchsize/gpus_per_trial) | Training Log/Training Command |
+|:-------:|:-----:|:-------:|:-------:|:-------:|
+| [Dogs vs. Cats](https://www.kaggle.com/c/dogs-vs-cats-redux-kernels-edition/data) | 0.17131/0.07326/0.03302/15%(196/1314) | ('resnet34_v1b', 'resnet34_v1', 'resnet34_v2')/(1e-4, 1e-2, log=True)/(0.86, 0.99)/(1e-6, 1e-3, log=True) | 180/30/384/4 | [log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/dogs-vs-cats-redux-kernels-edition/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/dogs.sh) |
+|[Aerial Cactus Identification](https://www.kaggle.com/c/aerial-cactus-identification/data)|0.9711/0.9999/1.0/12%|('resnet34_v1b')/(1e-4, 1e-2, log=True)/(0.88, 0.95)/(1e-6, 1e-4, log=True)|180/30/256/2|[log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/aerial-cactus-identification/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/aerial.sh) |
+|[Plant Seedlings Classification](https://www.kaggle.com/c/plant-seedlings-classification)|0.97607/0.98362/1.0/9%(77/833)|('resnet50_v1', 'resnet50_v1b', 'resnet50_v1c','resnet50_v1d', 'resnet50_v1s')/(1e-4, 1e-3, log=True)/(0.93, 0.95)/(1e-6, 1e-4, log=True)|120/30/96/2|[log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/plant-seedlings-classification/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/plant.sh) |
+|[The ature Conservancy Fisheries Monitoring](https://www.kaggle.com/c/the-nature-conservancy-fisheries-monitoring)|1.01974/0.90176/0.29535/30%(117/388)|('resnet50_v1')/(1e-4, 1e-2, log=True)/(0.85, 1.00)/(1e-6, 1e-4, log=True)|120/30/96/2 | [log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/fisheries_Monitoring/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/fish.sh) |
```

Comment:
nature

Reply:
@aptsunny could you send a PR to this repo for webdata https://github.com/zhanghang1989/AutoGluonWebdata

Reply:
Some webdata is constantly updated, I submit the other existing files.